android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/details/FamilyDetailsActivity.kt

```diff
@@ -151,12 +162,18 @@ class FamilyDetailsActivity : BaseMultiLanguageActivity() {
   }
 
   private fun removeFamilyMenuItemClicked(familyId: String) {
-    AlertDialogue.showConfirmAlert(
-      this,
-      R.string.confirm_remove_family_message,
-      R.string.confirm_remove_family_title,
-      { familyDetailViewModel.removeFamily(familyId = familyId) },
-      R.string.family_register_ok_title
+    startActivity(
+      Intent(this, RemoveFamilyQuestionnaireActivity::class.java).apply {
+        putExtras(
+          QuestionnaireActivity.intentArgs(
+            clientIdentifier = familyId,
+            formName = FamilyFormConstants.REMOVE_FAMILY
+          )
+        )
+        putExtra(FamilyQuestionnaireActivity.QUESTIONNAIRE_CALLING_ACTIVITY, getCallerActivity())
+        putExtra(QUESTIONNAIRE_ARG_PATIENT_KEY, familyId)
+        putExtra(QUESTIONNAIRE_ARG_FAMILY_NAME_KEY, familyName)
+      }
```

Comment:
@ellykits Do we have a Family object or patient object we can pass around the different screens on the intents? This way then we are sure we don't have to keep adding what we are passing in the intents. i.e 1 key gets us all the patient data.

Reply:
removed familyName and key passing to questionnaireActivity